overlord/fdestate/fdemgr.go

```diff
@@ -91,6 +91,11 @@ func (m *FDEManager) Ensure() error {
 
 // StartUp implements StateStarterUp.Startup
 func (m *FDEManager) StartUp() error {
```

Comment:
my preference would be to moved the first checks  here to Manager and set initErr already there for preseed and withouth SecbootSupport, then here we can just return nil if initErr != nil